### PR TITLE
First version of java template

### PR DESCRIPTION
**Pull request by ppolakiewicz**

### CODE REVIEW

File: playbot/java/src/main/java/com/RunBot.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com;
+
+public class RunBot {
+
+    public static void main(String[] args){
+        String host = "localhost";
+        int port = 21000;
```
Comment (devdo-eu) [inline playbot/java/src/main/java/com/RunBot.java]: is there a way to get host and port from args?
sorry but i don't really read in this lang ;)

File: playbot/java/src/main/resources/log4j2.properties [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+logger.rolling.level = debug
+logger.rolling.additivity = false
+logger.rolling.appenderRef.rolling.ref = RollingFile
+
+rootLogger.level=debug
+rootLogger.appenderRefs=stdout
+rootLogger.appenderRef.stdout.ref=STDOUT
+
+
+
```
Comment (devdo-eu) [inline playbot/java/src/main/resources/log4j2.properties]: too many empty lines.

File: playbot/java/src/main/java/com/PlayBot.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import com.enums.Move;
+import lombok.extern.log4j.Log4j2;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Random;
+
+@Log4j2
+public class PlayBot {
```
Comment (devdo-eu) [inline playbot/java/src/main/java/com/PlayBot.java:14]: I am not sure it is a time... but we need to document this codes.

**Review by devdo-eu — CHANGES_REQUESTED** [review]
I had some questions. :)

File: playbot/java/src/main/java/com/RunBot.java (hunk 1 above)
Comment (ppolakiewicz) [reply playbot/java/src/main/java/com/RunBot.java]: Sure there is :)
Just check the new version.

File: playbot/java/src/main/java/com/PlayBot.java (hunk 3 above)
Comment (ppolakiewicz) [reply playbot/java/src/main/java/com/PlayBot.java:14]: Will do after we finish beta test and have nearly final version :)

**Review by devdo-eu — APPROVED** [review]
Look good to me.

**Pull request merged**